### PR TITLE
Optimization image size. Remove install cache and docs

### PR DESCRIPTION
**Pull request by renko88r**

### CODE REVIEW

**Review by speter555 — COMMENTED** [review]
Please write this change to the release note

**Review by nagyzekkyandras — APPROVED** [review]
Seems good.

File: dockerfile/builder/galleon/Dockerfile
Comment (nagyzekkyandras) [inline dockerfile/builder/galleon/Dockerfile:1]: line 16 missing `&& \` from the end of the line

Comment (renko88r) [reply dockerfile/builder/galleon/Dockerfile:1]: tnx

**Review by rombow — APPROVED** [review]

File: dockerfile/builder/alpine-runner/Dockerfile [hunk 1]
```diff
@@ -4,12 +4,7 @@ FROM ${BASE_IMAGE} as base
 
 USER root
 
-RUN apk update --purge --no-cache && \
-    apk add --update --purge --no-cache bash && \
-    apk add --update --purge --no-cache git openssh && \
-    apk add --update --purge --no-cache curl && \
-    apk add --update --purge --no-cache libxml2-utils && \
-    apk add --update --purge --no-cache jq
+RUN apk add --purge --no-cache --no-interactive --no-progress bash git openssh curl libxml2-utils jq
```
Comment (bucherarnold) [inline dockerfile/builder/alpine-runner/Dockerfile:7]: az apk update ide már tuti nem kell?

**Review by bucherarnold — APPROVED** [review]

**Review by czenczl — APPROVED** [review]

**Review by cserhatit — APPROVED** [review]

**Review by vasizs — APPROVED** [review]

**Review by speter555 — APPROVED** [review]

**Pull request merged**